app/controllers/resource_evaluations_controller.rb

```diff
@@ -7,6 +7,12 @@ def create
                                                  resource_id:)
     resource_evaluation.evaluation = evaluation
     resource_evaluation.save if resource_evaluation.has_changes_to_save?
-    redirect_to(Resource.find(resource_id))
+    redirect_to_resource(resource_id)
+  end
+
+  private
+
+  def redirect_to_resource(resource_id)
```

Comment:
I think this is another reason to don't have this method. Is defined on two different files.